### ISSUE TITLE
cae:void_portal block removes all quest rewards when touched

### ISSUE THREAD

**Issue by PikachuGabe** [CoolerGangster/Create-Arcane-Engineering#87]
Doesn't report anything to any log, almost as if it were intentional. Has resulted in me being unable to receive any quest rewards in a server due to curiosity making me go through the portal without knowing anything about it(there was a warp, in this case). On the server I was playing on, it didn't affect an OP'd player, but it did affect me in both my creative test worlds as well as the server.

**Comment by PikachuGabe** [comment]
should be noted that the bug affected my teammates aswell on the server

**Comment by PikachuGabe** [comment]
after looking into why it happens, it seems that the portal block runs /ftbquests block_rewards true <playername> on the person who touches the portal, hiding all quest rewards for everyone in the team (all mods that mess with ftbteams run the command on the entire team not just one person)

**Comment by CoolerGangster** [comment]
Not a bug. You can Cheese the final trials with quest rewards. Of course having it affect all teammates isnt the most optimal solution , but it's a valid thing nonetheless.

**Comment by PikachuGabe** [comment]
hm, would it be possible to make a better warning then? currently it seems like the only warning is a lectern outside the portal, however the lectern can easily be broken on servers if someone wants to troll.

**Comment by CoolerGangster** [comment]
Theres the red large text when you enter the dimension

**Comment by PikachuGabe** [comment]
yes but which portal activates that text? cae:void_portal or... whichever one takes you to the void portal, i dont remember which it is.

**Comment by CoolerGangster** [comment]
Dont know right now either
